lib/utils/index.js

```diff
@@ -273,6 +273,48 @@ module.exports = {
   },
 
   /**
+   * Gets the property name of a given node.
+   * @param {ASTNode} node - The node to get.
+   * @return {string|null} The property name if static. Otherwise, null.
+   */
+  getStaticPropertyName (node) {
```

Comment:
this method is not same but almost, and ast-utils is not exported than i will have to import file directly what can caus some errors between versions of eslint.